### DISCUSSION TITLE
Welcome to mautic-whitelabeler Discussions!

### DISCUSSION THREAD

**Discussion by nickian** [nickian/mautic-whitelabeler#157] [Announcements]
## 👋 Welcome!

Use this space to discuss any issues or feature requests. If you're having trouble getting the whitelabeler to work in your hosting environment, ask for help here. Don't create an issue in the repository. Issues are reserved for bugs or features related directly to the whitelabeler, not Mautic itself or your hosting environment.

You can also join the new public chat server here: [Mautic Chat](https://creatv.link/mautic-chat)

**Comment by Tahor** [comment]
Any chance to get an update on this? When I tried installing on [5.2.1] it was infinitely loading.
